app/tests/test_search.py

```diff
@@ -101,6 +103,8 @@
     }
 }
 
+expected_file_name = "fifth_file.doc"
```

Comment:
can we just put this in each test that needs it? I feel like in tests unless it was unwieldy duplication its nicer to have all the context in there to understand the test